src/backend/src/controllers/projects.controllers.ts

```diff
@@ -102,9 +102,10 @@ export default class ProjectsController {
     try {
       const user: User = await getCurrentUser(res);
       const wbsNumber: WbsNumber = validateWBS(req.params.wbsNum);
+      const { identifier } = req.body;
       const organizationId = getOrganizationId(req.headers);
 
-      const deletedProject: Project = await ProjectsService.deleteProject(user, wbsNumber, organizationId);
+      const deletedProject: Project = await ProjectsService.deleteProject(user, wbsNumber, identifier, organizationId);
```

Comment:
For references to the identifier, call it changeRequestIdentifier to not be ambiguous